docs/index.md

```diff
@@ -24,7 +24,7 @@ You are currently looking at the **Tutorials and Guides** for Sequelize. You mig
 ## Quick example
 
 ```js
-const { Sequelize, Model, DataTypes } = require('sequelize');
+const { Sequelize, Model, DataTypes } = require("@sequelize/core");
```

Comment:
We're using single quotes in the codebase so we should make our examples conform to this as well. Some examples are using double quotes but it would be nice if you could replace those as well. I started with making suggestions in the review manually but that was not very doable